pom.xml

```diff
@@ -158,8 +158,8 @@
     <scala.version>2.11.12</scala.version>
     <scala.binary.version>2.11</scala.binary.version>
     <codehaus.jackson.version>1.9.13</codehaus.jackson.version>
-    <fasterxml.jackson.version>2.6.7</fasterxml.jackson.version>
-    <fasterxml.jackson.databind.version>2.6.7.1</fasterxml.jackson.databind.version>
+    <fasterxml.jackson.version>2.9.6</fasterxml.jackson.version>
+    <fasterxml.jackson.databind.version>2.9.6</fasterxml.jackson.databind.version>
```

Comment:
I suspect we can collapse these two versions; they were broken out to handle the fact that a few 2.6.x Jackson releases didn't publish all artifacts.

Reply:
Good point, I've merged them